app/src/main/java/com/example/activeamigo/ForgotPassword.java

```diff
@@ -0,0 +1,91 @@
+package com.example.activeamigo;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.ActionBar;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.view.MenuItem;
+import android.widget.EditText;
+
+import com.google.android.gms.tasks.Task;
+import com.google.android.gms.tasks.TaskCompletionSource;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.QuerySnapshot;
+
+public class ForgotPassword extends AppCompatActivity implements Alertable{
+   private FirebaseAuth auth;
+   private FirebaseFirestore db;
+   private EditText emailText;
```

Comment:
consistency convention: in login you set them null

Reply:
Initialized to null